src/javascript/binary/base/page.js

```diff
@@ -398,11 +398,14 @@ Header.prototype = {
     on_load: function() {
         this.show_or_hide_login_form();
         this.register_dynamic_links();
-        if (!this.clock_started) this.start_clock();
+        //if (!this.clock_started) this.start_clock();
+        if (!this.clock_started) this.start_clock_ws();
+        //start_clock_ws
         this.simulate_input_placeholder_for_ie();
     },
     on_unload: function() {
         this.menu.reset();
+        if (!this.clock_started) this.start_clock_ws();
```

Comment:
Because at the moment, our WS ready status is often false onLoad . Its however true on unLOad. Should it be true on load, our timer would start and it would not be started again on unload, else our time would start on unload or page navigation. Either of the two, one does not cancel the other. :)